sites/partners/src/lib/applications/formatApplicationData.ts

```diff
@@ -85,42 +85,30 @@ export const mapFormToApi = ({
     ? data.application?.language
     : null
 
-  const submissionDate: Date | null = (() => {
-    // rename default (wrong property names)
-    const {
-      day: submissionDay,
-      month: submissionMonth,
-      year: submissionYear,
-    } = data.dateSubmitted || {}
-    const { hours, minutes = 0, seconds = 0, period } = data?.timeSubmitted || {}
+  const getDateTime = (date: DateFieldValues, time: TimeFieldValues) => {
+    let day = date?.day
+    let month = date?.month
+    const year = date?.year
+    const { hours, minutes = 0, seconds = 0, period } = time || {}
+
+    if (!day || !month || !year) return null
 
-    if (!submissionDay || !submissionMonth || !submissionYear) return null
+    if (month.length === 1) month = `0${month}`
```

Comment:
**note:** This is the only change - the rest was just refactoring to reduce duplication